tests/pytests/reg/test_uvm_reg_model.py

```diff
@@ -5,6 +5,7 @@
 pytestmark = pytest.mark.usefixtures("initialize_pyuvm")
 
 
+@pytest.mark.skip(reason="Not implemented")
```

Comment:
These should definitely be
```suggestion
@pytest.mark.xfail(raises=NotImplementedError)
```

This will ensure if the functionality is added in the future this test will start failing, signaling it should have the mark removed and the test re-evaluated for correctness.